corgi/core/files.py

```diff
@@ -67,11 +67,12 @@ class ProductManifestFile(ManifestFile):
     def render_content(self) -> str:
 
         latest_components = self.obj.get_latest_components()  # type: ignore[attr-defined]
```

Comment:
Is this unused now then?

Reply:
Ah, fail, I didn't realize released_ was being run on top of latest_. Ignore me! :)